<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: white;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="playerMoveImage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="botMoveImage">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scissorsButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(250, 250, 250);
	background-color: rgb(130, 170, 90);
	padding-top: 5px;
	border: none;
	border-radius: 10px;
	border-left: 1px solid rgb(110, 144, 76);
	border-right: 1px solid rgb(110, 144, 76);
	border-bottom: 5px solid rgb(110, 144, 76);
}

QPushButton:hover {
	background-color: rgb(138, 181, 96);
	border-left: 1px solid rgb(110, 144, 76);
	border-right: 1px solid rgb(110, 144, 76);
	border-bottom: 1px solid rgb(110, 144, 76);
}

QPushButton:pressed{
	background-color: rgb(120, 157, 83);
	padding-top: 5px;
	border-left: 1px solid rgb(110, 144, 76);
	border-right: 1px solid rgb(110, 144, 76);
	border-top: 5px solid rgb(110, 144, 76);
	border-bottom: none;
}</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Scissors</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rockButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(250, 250, 250);
	background-color: rgb(255, 180, 18);
	padding-top: 5px;
	border: none;
	border-radius: 10px;
	border-left: 1px solid rgb(255, 140, 18);
	border-right: 1px solid rgb(255, 140, 18);
	border-bottom: 5px solid rgb(255, 140, 18);
}

QPushButton:hover {
	background-color: rgb(255, 200, 18);
	border-left: 1px solid rgb(255, 140, 18);
	border-right: 1px solid rgb(255, 140, 18);
	border-bottom: 1px solid rgb(255, 140, 18);
}

QPushButton:pressed{
	background-color: rgb(255, 180, 18);
	padding-top: 5px;
	border-left: 1px solid rgb(255, 140, 18);
	border-right: 1px solid rgb(255, 140, 18);
	border-top: 5px solid rgb(255, 140, 18);
	border-bottom: none;
}</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Rock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paperButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(250, 250, 250);
	background-color: rgb(255, 180, 255);
	padding-top: 5px;
	border: none;
	border-radius: 10px;
	border-left: 1px solid rgb(255, 128, 255);
	border-right: 1px solid rgb(255, 128, 255);
	border-bottom: 5px solid rgb(255, 128, 255);
}

QPushButton:hover {
	background-color: rgb(255, 190, 255);
	border-left: 1px solid rgb(255, 110, 255);
	border-right: 1px solid rgb(255, 110, 255);
	border-bottom: 1px solid rgb(255, 110, 255);
}

QPushButton:pressed{
	background-color: rgb(255, 160, 255);
	padding-top: 5px;
	border-left: 1px solid rgb(255, 128, 255);
	border-right: 1px solid rgb(255, 128, 255);
	border-top: 5px solid rgb(255, 128, 255);
	border-bottom: none;
}</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Paper</string>
    </property>
   </widget>
   <widget class="QLabel" name="roundResultText">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="winsScore">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Wins : </string>
    </property>
   </widget>
   <widget class="QLabel" name="losesScore">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Loses : </string>
    </property>
   </widget>
   <widget class="QLabel" name="drawsScore">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Draws : </string>
    </property>
   </widget>
   <widget class="QLabel" name="playerName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>You</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="botName">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Bot</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="roundsPlayed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rounds played : </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
